docs/pages/includes/role-spec.mdx

```diff
@@ -236,7 +236,9 @@ spec:
       # the `claims_to_roles` mapping works the same as it does in
       # the OIDC connector, with the added benefit that the roles being mapped to
       # can also be matchers. the below mapping says that users with
-      # the claims `groups: admins` can request any role in the system.
+      # claims that match to `projects: project-(.*)` can request
+      # roles that match with '$1-admin' with the first match group.
```

Comment:
Is there a difference between "match to" and "match with"? 

If not, I would use consistent language. 

Probably also worth mentioning that these are regular expressions too.

Reply:
thanks, update